docs/Data Provider Nodes/flightaware-chainlink-testnet.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "FlightAware Chainlink (Testnet)"
+permalink: "docs/flightaware-chainlink-testnet/"
+hidden: true
+---
+This Chainlink has a dedicated connection to [FlightAware](https://uk.flightaware.com/).
+
+# Steps for using this oracle
+
+- Write and deploy your [Chainlink](../request-and-receive-data/) contract using the network details below
+- Fund it with [LINK](../link-token-contracts/)
+- Call your [request method](./#chainlink-examples)
+
+# Chainlink Network Details
+
+You will need to use the following LINK token address, oracle address, and Job IDs to create the request.
+
+#### Kovan
+LINK Token address: `0xa36085f69e2889c224210f603d836748e7dc0088`
+Oracle address: `0x2f90A6D021db21e1B2A077c5a37B3C7E75D15b7e`
+JobID: `a644d4e30977459d9a596bef89c09e71 `
+Sleep JobID: `f0003b2c52024e7fa931d6ee9a947c87`
+
+# Create your Chainlink contract
+
+Import `ChainlinkClient.sol` into your contract so you can inherit the Chainlink behavior.
+
+```javascript Solidity 4
```

Comment:
Mark this as a solidity block rather than `javascript`

Reply:
Mark updated